src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -89,8 +90,9 @@ export const getModuleDetails = async (
   // Create ancillary data for proposal hash
   let ancillaryData = '';
   let timestamp = 0;
+  let proposalHash;
   if (transactions !== undefined) {
-    const transactionsHash = keccak256(
+    proposalHash = keccak256(
```

Comment:
If transactions are undefined, we can't really do anything else in this method, can we? All these variables we're setting in here are absolutely necessary for the rest of the method. Should we just return early if transactions are undefined, so we don't run into unexpected behavior or errors?

Reply:
That's a good point. If there are no transactions specified, we could have a state that doesn't show any proposal or execution buttons. That would mean the Snapshot interface is just voting for signaling purposes and there is nothing executable on-chain.

Reply:
I'll do that with the follow-up PR, where I'll also add a step that asks the user if the vote was affirmative before showing them the button to propose.